users_template.dat

```diff
@@ -1,4 +1,6 @@
-# User data base of the silly user system.
+# User data base of the development user system.
 # THIS DATA IS NOT SECURE.
 # DO NOT USE FOR STORING IMPORTANT PASSWORDS
 #
+# One line per user:
+# username:password
```

Comment:
plaintext password? Even for development, that feels a bit icky in these days. Why not hash it? `htpasswd` or something?

Reply:
i see little point in changing that